recipes/multiqc/meta.yaml

```diff
@@ -14,6 +14,7 @@ build:
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
Could you also add `setuptools` to the run requirements and add
```yaml
  skip-lints:
    - uses_setuptools  # uses pkg_resources
```
to the `extra` section?

Reply:
Does it actually need `setuptools` to run? That'd be rather unusual.

Reply:
Yes, it uses the `pkg_resources` package which is part of `setuptools`, see: https://github.com/ewels/MultiQC/search?q=pkg_resources. But you are right, most packages do not require `setuptools` to run. The usage of `pkg_resources` is one of those special cases which make it a requirement, though.